Framework/LiveData/src/FileEventDataListener.cpp

```diff
@@ -41,14 +43,15 @@ FileEventDataListener::FileEventDataListener()
     } else {
       auto loader = FileLoaderRegistry::Instance().chooseLoader(m_filename);
       m_loaderName = loader->name();
-      if (m_loaderName.find("Nexus") != std::string::npos &&
-          (m_loaderName.find("Pre") != std::string::npos || m_loaderName.find("Event") != std::string::npos)) {
-        if (m_loaderName.find("Pre") != std::string::npos && m_loaderName.find("Event") != std::string::npos) {
-          m_filePropName = "EventFilename";
-          m_canLoadMonitors = false;
-        }
+      if (m_loaderName == "LoadEventPreNexus") {
+        m_filePropName = "EventFilename";
+        m_canLoadMonitors = false;
+      } else if (m_loaderName == "LoadEventNexus") {
+        m_filePropName = "Filename";
+        m_canLoadMonitors = true;
       } else {
-        g_log.error("No loader for " + m_filename + " that supports chunking. The algorithm will fail.");
+        g_log.error("No loader for " + m_filename + " that supports chunking (found loader '" + m_loaderName +
```

Comment:
Would this be better to be an exception rather than a log message? Since it is mostly developers that would use this, they are likely to notice the error log once it the workflow fails so this isn't a necessary change.